pyxem/generators/indexation_generator.py

```diff
@@ -37,7 +37,13 @@
     select_method_from_method_dict,
     transfer_navigation_axes,
 )
+from pyxem.utils._deprecated import deprecated
 
+
+@deprecated(since="0.14",
+            alternative="pyxem.generators.AcceleratedIndexationGenerator or pyxem.generators.VectorIndexationGenerator",
+            alternative_is_function=True,
+            removal="1.0.0", )
```

Comment:
This decorator is only for wrapping functions and class methods, not entire classes (hence the failing indexation generator test). I think this decorator is unnecessary since it's not possible to create a generator anyway due to the value error.